pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -141,6 +141,7 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 	if err != nil {
 		return api.Environment{}, err
 	}
+	overrideKafkaTopicsEnv(cr, &mergedEnv)
```

Comment:
I think it should be inside getKafkaConfig. isn't it better/;

Reply:
this needs to be explored more